cvat_proxy/conf.d/cvat.conf.template

```diff
@@ -12,24 +12,11 @@ server {
     proxy_set_header        Host $http_host;
     proxy_pass_header       Set-Cookie;
 
-    location ~* /api/.*|git/.*|analytics/.*|static/.*|admin|admin/.*|documentation/.*  {
+    location ~* /api/.*|git/.*|analytics/.*|static/.*|admin(?:/(.*))?.*|documentation/.*|django-rq(?:/(.*))? {
```

Comment:
@azhavoro , I believe we have a duplication of the file in documentation. Could you please update all places?